desktop-exporter/app/main.css

```diff
@@ -0,0 +1,58 @@
+html,
+body {
+  height: 100%;
+  margin: 0px;
+  overflow: hidden;
+}
+
+.container {
+  display: flex;
+}
+
+.sidebar {
+    width: 250px;
+    transition: width 0.3s ease-in-out;
+    height: 100vh;
+    background-color: #ffaaa5;
+    padding-top: 30px;
+}
+
+.sidebar.closed {
+    transition: width 0.3s ease-in-out;
+    width: 50px
+}
+
+.traceview {
+  height: 100vh;
+  width: 100vw;
+  display: grid;
+  grid-gap: 0px;
+  grid-template-columns: 3fr 200px;
+  grid-template-rows: 200px 1fr;
+  grid-template-areas:
+    "header  header"
```

Comment:
Just so you know I'm reading this like 
![Badgers_Badgers](https://user-images.githubusercontent.com/175496/203433988-6964b3c0-98b9-43f1-9c7d-ee3fb78bb5d5.gif)

Reply:
Sidebar! Sidebar! 🍄 🍄